src/ServiceInsight/Explorer/EndpointExplorer/EndpointExplorerViewModel.cs

```diff
@@ -3,203 +3,97 @@
     using System;
     using System.Linq;
     using Caliburn.Micro;
-    using ServiceInsight.ExtensionMethods;
-    using ServiceInsight.Framework;
-    using ServiceInsight.Framework.Events;
-    using ServiceInsight.Framework.Settings;
-    using ServiceInsight.ServiceControl;
-    using ServiceInsight.Settings;
-    using ServiceInsight.Shell;
-    using ServiceInsight.Startup;
-
-    public class EndpointExplorerViewModel : Screen,
-        IHandle<RequestSelectingEndpoint>
+    using EndpointExplorer;
+    using Framework;
+    using Framework.Events;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+
+    public class EndpointExplorerViewModel : Screen, IHandle<RequestSelectingEndpoint>, IHandle<SelectedExplorerItemChanged>
     {
-        IEventAggregator eventAggregator;
-        ISettingsProvider settingsProvider;
-        IServiceControl serviceControl;
-        NetworkOperations networkOperations;
-        ServiceControlConnectionProvider connectionProvider;
-        CommandLineArgParser commandLineParser;
-
-        public EndpointExplorerViewModel(
-            IEventAggregator eventAggregator,
-            ISettingsProvider settingsProvider,
-            ServiceControlConnectionProvider connectionProvider,
-            CommandLineArgParser commandLineParser,
-            IServiceControl serviceControl,
-            NetworkOperations networkOperations)
+        static JsonSerializer serializer = new JsonSerializer();
+
+        static EndpointExplorerViewModel()
         {
-            this.eventAggregator = eventAggregator;
-            this.settingsProvider = settingsProvider;
-            this.serviceControl = serviceControl;
-            this.networkOperations = networkOperations;
-            this.connectionProvider = connectionProvider;
-            this.commandLineParser = commandLineParser;
-            Items = new BindableCollection<ExplorerItem>();
+            serializer.ContractResolver = new SnakeCasePropertyNamesContractResolver();
```

Comment:
I find the split a bit confusing...could this be merged with the initialiation code above? e.g.

    static EndpointExplorerViewModel()
    {
        serializer = new JsonSerializer { ContractResolver = new SnakeCasePropertyNamesContractResolver() }; 
    }